File: app/src/main/java/fmrsabino/moviesdb/ui/search/SearchMvpView.java
package fmrsabino.moviesdb.ui.search;

import fmrsabino.moviesdb.data.model.search.Search;
import fmrsabino.moviesdb.ui.base.MvpView;

public interface SearchMvpView extends MvpView {
    void showSearchResults(Search search, boolean newResults);
    void getPosterImageUrl(String posterPath);
}

File: app/src/main/java/fmrsabino/moviesdb/ui/search/SearchMvpView.kt
package fmrsabino.moviesdb.ui.search

import fmrsabino.moviesdb.data.model.search.Search
import fmrsabino.moviesdb.ui.base.MvpView

interface SearchMvpView : MvpView {
    fun showSearchResults(search: Search, newResults: Boolean)
    fun getPosterImageUrl(posterPath: String)
}
